src/main/kotlin/lotto/data/LottoRanking.kt

```diff
@@ -0,0 +1,33 @@
+package lotto.data
+
+enum class LottoRanking(val matchingNumberCnt: Int, val price: Int) : Prize {
+    FirstPlace(6, 2000000000) {
```

Comment:
```suggestion
    FirstPlace(6, 2_000_000_000) {
```

언더바(`_`)를 통해 가독성을 높혀줄 수 있습니다 ㅎㅎ